Neos.Flow/Classes/Cli/Response.php

```diff
@@ -47,12 +47,12 @@ class Response
     private $content = '';
 
     /**
-     * @var
+     * @var boolean|int
```

Comment:
```suggestion
     * @var bool|int
```

PSR-12:
> Short form of type keywords MUST be used i.e. bool instead of boolean, int instead of integer etc.
https://www.php-fig.org/psr/psr-12/#25-keywords-and-types